.travis.yml

```diff
@@ -1,20 +1,30 @@
 language: python
 sudo: required
+os: linux
+cache:
+  pip: true
 matrix:
   include:
-  - python: '3.6'
-  - python: '3.7'
-    dist: xenial
-  - os: windows
-    language: bash
-    python: '3.7'
-    before_install:
-      - export TRAVIS_PYTHON_VERSION=3.7
-      - choco install python --version $TRAVIS_PYTHON_VERSION
-      - export PATH="/C/Python37:/C/Python37/Scripts:$PATH"
+    - python: "3.6"
```

Comment:
this was unnecessary -- I solved it in #236 with the enhancement in `noxfile.py`